_data/projects/kodadot.yml

```
@@ -0,0 +1,11 @@
+---
+name: KodaDot - Polkadot NFT marketplace
+desc: open source implementation of nine NFT standards and growing to become multi-chain available nft marketplace
+site: http://github.com/kodadot/nft-gallery
+tags:
+- polkadot
+- vuejs
+- vue3
+upforgrabs:
+  name: good first issue
+  link: https://github.com/kodadot/nft-gallery/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22
```

:art: a simpler URL that will help with generating stats on our end

```suggestion
  link: https://github.com/ForNeVeR/Cesium/labels/good-first-issue
```